2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -75,6 +77,6 @@ private void configureButtonBindings() {
    */
   public Command getAutonomousCommand() {
     // An ExampleCommand will run in autonomous
-    return m_autoCommand;
+    return null;
```

Comment:
why

Reply:
I assumed this was an artifact of the merge commit that @jaredgoodman03 made. While it is innocuous (as `m_autoCommand` was removed), we should avoid making changes like these in the future, as this change will be marked on the blame as coming from the drivetrain commit + it makes the history confusing in general. These cleanup changes should be made in a separate PR.